src/inverseRobot.js

```diff
@@ -23,8 +23,25 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  const robotData = Object.entries(robot);
+
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < (robotValues.length - 1); i++) {
+    if (robotValues[i] === robotValues[i + 1]) {
```

Comment:
you just return null if the value the same as the next one, lets try to solve this without robotData and robotValue arrays and using only one loop.
*hint: you need to use that kind of loop which can give access to the object key

Reply:
Thanks for the good advice!